# Fix an error from using a non-standard Linux header file.
# This patch was developed in-house and is suitable for upstream use.
#
--- src/tspi/tsp_tcsi_param.c
+++ src/tspi/tsp_tcsi_param.c
@@ -15,6 +15,9 @@ #include <stdio.h>
 
 #ifndef __APPLE__
 #include <limits.h>
+#if defined (SOLARIS)
+#define HOST_NAME_MAX _POSIX_HOST_NAME_MAX
+#endif
 #else
 #define HOST_NAME_MAX 64
 #endif
